src/Ethereum.sol

```diff
@@ -16,8 +32,16 @@ library Ethereum {
     /*** SparkDAO Addresses                                                                                         ***/
     /******************************************************************************************************************/
 
-    address internal constant META_MORPHO_VAULT = 0x73e65DBD630f90604062f6E02fAb9138e713edD9;
-    address internal constant SPARK_PROXY      = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
+    address internal constant SPARK_PROXY = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
+
+    /******************************************************************************************************************/
+    /*** Morpho Addresses                                                                                           ***/
+    /******************************************************************************************************************/
+
+    address internal constant MORPHO_DEFAULT_IRM      = 0x870aC11D48B15DB9a138Cf899d20F13F79Ba00BC;
+    address internal constant MORPHO_STETH_ETH_ORACLE = 0x86392dC19c0b719886221c78AB11eb8Cf5c52812;
+    address internal constant MORPHO_VAULT_DAI_1      = 0x73e65DBD630f90604062f6E02fAb9138e713edD9;
+    address internal constant MORPHO_WBTC_BTC_ORACLE  = 0xfdFD9C85aD200c506Cf9e21F1FD8dd01932FBB23;
```

Comment:
These are Chainlink oracles and are for the kill switch. If you want to add the sUSDe and USDe oracles, they are `0x5D916980D5Ae1737a8330Bf24dF812b2911Aae25` and `0xaE4750d0813B5E37A51f7629beedd72AF1f9cA35` respectively.